### ISSUE TITLE
Settings pane has inverted background coloring

### ISSUE THREAD

**Issue by rjv** [silvestreh/atom-material-syntax#24]
After updating to v0.3.0, the settings pane has an entirely white background. I'm using the standard Atom Material UI theme and syntax theme.

![20160128_123800_screenshot](https://cloud.githubusercontent.com/assets/532439/12653174/46b0b5bc-c5bc-11e5-8bf6-821b2de1b691.png)

**Comment by silvestreh** [comment]
That's an interesting one… it's probably related to the UI theme as `0.8.0` doesn't explicitly define a background color for the settings panes.

I know this was introduced with the latest release (which I probably shouldn't have pushed). I changed a few things to make the syntax theme play better with the development version of the UI theme.

I decided to no longer support the UI theme `0.8.0` as the development version is a completely different codebase. However, I'll make an exception and try to fix this for you.

If you wish to test the UI theme's development version (I need testers), open a terminal and run:

```shell
apm uninstall atom-material-ui
git clone git@github.com:silvestreh/atom-material-ui.git
cd atom-material-ui/
apm link && apm install
```

Cheers!
